Configuration/Eras/python/Modifier_run2_miniAOD_pp_on_AA_103X_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+run2_miniAOD_pp_on_AA_103X =  cms.Modifier()
```

Comment:
please move this to Configuration/ProcessModifier, make it similar to `run2_miniAOD_UL`

Reply:
I moved it Configuration/ProcessModifiers, but now it's not picked up from the driver flag --era Run2_2018_pp_on_AA,run2_miniAOD_pp_on_AA_103X

Reply:
You have to use the `--procModifier` flag